content/guides/how-to-buy-and-sell-a-star.md

```diff
@@ -0,0 +1,140 @@
+
+By the way, the star also has an Ethereum address, so it can hold funds and so on. This address is deterministically generated from the current owner's address. More on this below when we talk about the Master Ticket.
+
+### How to See How Many Planets a Star Has Spawned
+
+There are two good ways to check how many planets a star has spawned, and therefore how many stars it has available to spawn.
+
+The first and simplest is to open [this giant list of events](https://gaze-exports.s3.us-east-2.amazonaws.com/events.txt) generated by the Azimuth tool.
+
+Every time a planet or star has changed hands, or breached, or a star has spawned a planet, the event is recorded here. You can search for either the name of the star, or its Ethereum address, with CMD-F or CTRL-F (however you search for text on web pages). Searching for either the name or for the address of the star will find it.
+
+For instance, I own a planet spawned by ~marnus. I can search for "marnus,spawned" and see every time ~marnus has spawned a star. Or I could search for "~marnus" and see every action ~marnus has taken on the network. If I wanted to buy the star ~marnus, I could search for its name and simply count how many planets it had spawned.
+
+If you have the star's Ethereum contract address, rather than that of its owner, there is a second way to check how planets a star has spawned. Go to [Etherscan](https://etherscan.io/address/azimuth.eth#readContract). There, under Contract, click Read Contract, and you will see a long list of possible attributes of the star you can check:
+        - 1. escapeRequestsIndexes
+        - 2. getOwnedPoints
+        - ...
+        - 65. isRequestingEscapeTo
+
+Of the many possible attributes you could check, you will want to look at `16. getSpawnCount`, and `32. points`. These should both read `0`.
+
+### Taking Possession of Your Star
+
+Now that you have bought a star, how do you get control of it? There are two main possibilities, and both involve Bridge, at [bridge.urbit.org](https://bridge.urbit.org/). 
+
+First, you might have been sent the Master Ticket. This contains a username and a long passphrase. Use these to log in to Bridge. At this point, the seller still owns your star. You should immediately transfer ownership to your own wallet. You can then generate a new Master Ticket, or leave that till later. The old Master Ticket will be invalid as soon as you transfer ownership. You now control your star.
+
+Second, when you purchased your star, the owner may have transferred ownership to your wallet address. This is a little more likely, because it's easy. Go to Bridge, and choose to log in using the wallet address. You will simply sign a transaction which verifies that you control the address which controls this star. You now control your star. Again, you may want to generate a Master Ticket, although you don't need to because no one else has this information.
```

Comment:
```suggestion
Second, when you purchased your star, the owner may have transferred ownership to your Ethereum wallet. Go to [Bridge](https://bridge.urbit.org), and connect your wallet. Metamask is the easiest wallet to use, but a number of wallets which support WalletConnect work as well. If you're received your star in a wallet that doesn't support WalletConnect, you can also login with the wallet's seedphrase or set it up in MetaMask. If the star has been directly transferred, it will be listed among the Urbit IDs you own as soon as you log in. Alternatively, you may see an incoming transfer of the star. In that case, you will need some Ethereum in the wallet to submit a transaction accepting the transfer. Once accepted, it will appear among the Urbit IDs you own.
```